tests/browser/heda_settings_accounts_and_contacts_edit_mode.robot

```diff
@@ -0,0 +1,92 @@
+*** Settings ***
+
+Resource        tests/HEDA.robot
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+
+Go To Heda Home
+    Go To Heda Home
+    Select Frame			//iframe[contains(@id, "vfFrameId")]
+    Click Link				//a[contains(text(),'Accounts and Contacts')]
+    Wait Until Element Is visible	//a[contains(text(),'Accounts and Contacts')]
+    Capture Page Screenshot
+
+Validate Edit Mode For Accounts and Contacts, Settings
+    
+    
+    # Go To Heda Home
+    # Select Frame                    //iframe[contains(@id, "vfFrameId")]
+    # Click Element                   //div[@data-aura-rendered-by='106:0']//input[@type='radio']
+    # Capture Page Screenshot
+
+    #Go into Edit mode 
+    Click Element                   //div[@class='slds-button-group']//span[contains(text(), 'Edit')]
```

Comment:
This should be done in a generic python based keyword.  Generally, having locators in test cases should be avoided if possible.  This seems like a good case for a more generic keyword that can be reused in our tests (possibly in the Salesforce library if it's just interacting with Salesforce's UI rather than anything HEDA specific).

Reply:
Correct.  If ok, I will need to create a WI and make these cosmetic changes in a 2nd revision.